server/routes/category.js

```diff
@@ -32,6 +34,21 @@ router.get('/getAll', async (req, res) => {
         res.status(500).json({message: error.message})
     }
 })
+
+// Get by Name Method
+router.get('/getOne/:name', async (req, res) => {
```

Comment:
Echoing Will's comment here, we don't want 2 routes with the same name here. Since we're already prefixing these routes in the index.js file, see [here](https://github.com/TritonSE/SDCTA-Data-Visualization-Website/blob/main/server/index.js#:~:text=app.use(%22/api,visualization%22%2C%20routesVisualization)%3B), we can just name these / and then use a url param to differentiate whether we want to get by name or category. 

Request params will be stored in req.params. So we can make different methods in the service layer. We already have getCategoryByName, we can add a getCategoryById that just calls "await Model.findById(req.params.id);"